tests/detabler/detabler.js

```diff
@@ -0,0 +1,187 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: contextmenu,detabler */
+
+(function () {
+	'use strict';
+	bender.editor = {
+		config: {
+			enterMode: CKEDITOR.ENTER_P
+		}
+	};
+	bender.test({
+		assertHtml: function (expected, actual, msg) {
+			assert.areEqual(bender.tools.fixHtml(expected), bender.tools.fixHtml(actual), msg);
+		},
+		setUp: function () {
+			this.command = this.editorBot.editor.getCommand('detable');
+		},
+		'it does not do anything outside of tables': function () {
+			var editor = this.editorBot.editor,
+				startHtml,
+				endHtml;
+
+			startHtml = '<p>foobar^</p>' +
+				'<p>baz</p>' +
+				'<table>' +
+					'<tbody>' +
+						'<tr><td>leave me alone</td></tr>' +
+					'</tbody>' +
+				'</table>';
+
+			endHtml = '<p>foobar</p>' +
+				'<p>baz</p>' +
+				'<table>' +
+					'<tbody>' +
+						'<tr><td>leave me alone</td></tr>' +
+					'</tbody>' +
+				'</table>';
```

Comment:
the "^" cursor helper only works in the test helper for setting editor state.

Checks for what is currently in the editor uses editor.getData() which returns a string of the current HTML.  doesn't care at all about selection 

This test is basically just showing that nothing happens to the html if you try to run the command in a paragraph. This shouldn't happen in real life since the context menu item won't appear, but technically you can manually add a toolbar button to call the command, or call it via script, etc.